neurodegeneration/processing-containers/normative_biomarker_visualizer/files/normative_biomarker_visualizer.py

```diff
@@ -239,6 +250,10 @@ def plotWithRef(dfRef, WMLSref, dfSub, fname, spareAD, spareBA):
 			fig.append_trace( go.Scatter(mode='lines', x=XX_AD, y=XX10_AD, legendgroup='AD', marker=dict(color='MediumBlue', size=10), name='10th percentile for AD', line=dict(dash = 'dash'), showlegend=sl),row,column)
 			fig.append_trace( go.Scatter(mode='markers', x=dfSub.Age.tolist(), y=dfSub[selVar].tolist(), legendgroup='Patient', marker=dict(color='Black', symbol = 'circle-cross-open', size=16,line=dict(color='MediumPurple', width=3)), name='Patient', showlegend=sl),row,column)
 
+		# min_ = min(ADRef.Age.min(), CNRef.Age.min())
+		# max_ = min(ADRef.Age.max(), CNRef.Age.max())
+		# fig.update_xaxes(range=[min_, max_])
+  
```

Comment:
Please remove commented code. If it is commented for a specific reason, please add a comment about why it is there.

Reply:
done